torchgeo/datasets/landcoverai.py

```diff
@@ -86,6 +89,10 @@ def __init__(
             RuntimeError: if ``download=False`` and data is not found, or checksums
                 don't match
         """
+        if isinstance(root, list):
+            # TODO: Workaround until list are fully implemented
```

Comment:
I'm fine with whatever workarounds we need for these special cases for now.

Reply:
All right. Can we merge this and start a new PR for VectorDataset?

Reply:
I'm fine with doing VectorDataset in a separate PR. Just need to get all tests passing on this PR. Let me know if/when you want me to take over.